{"cve_repaire_en": "", "cve_code_exp_url": "", "cve_id": "CVE-2018-18385", "cve_categories": ["category1", "category2"], "cve_code_detect_url": "", "cve_name": "cve_name", "cve_risk_level": "determine", "cve_detail_en": "Asciidoctor in versions < 1.5.8 allows remote attackers to cause a denial of service (infinite loop). The loop was caused by the fact that Parser.next_block was not exhausting all the lines in the reader as the while loop expected it would. This was happening because the regular expression that detects any list was not agreeing with the regular expression that detects a specific list type. So the line kept getting pushed back onto the reader, hence causing the loop.", "cve_name_en": "cve_name_en", "edit_time": "Sat Oct 12 14:43:42 2019", "cve_code_des_url": "", "cve_other_info": "", "modify_time": "2018-10-23", "cve_detail": "", "references": [{"URL": "https://github.com/asciidoctor/asciidoctor/issues/2888", "Description": "MISC:https://github.com/asciidoctor/asciidoctor/issues/2888"}], "cve_repaire": "", "editor": "warcup", "add_time": "2018-10-16"}